packages/web/docs/src/components/masking-scrollview.tsx

```diff
@@ -0,0 +1,159 @@
+import { useEffect, useLayoutEffect, useReducer, useRef, useState } from 'react';
+
+export type MaskingScrollviewFade = {
+  y?: boolean;
+  x?: boolean;
```

Comment:
let's make it required and just provide `false`


```suggestion
  y: boolean;
  x: boolean;
```

Reply:
I'm actually thinking this might be `"x" | "y"`, because the mask gradients combined at the corners look sorta off.

Reply:
Oh yeah, building for both `x` and `y` would need a different mask gradient.

<img width="425" alt="image" src="https://github.com/user-attachments/assets/6517a3af-ce05-445a-990b-f7fda9dc0e5a">